packages/space-opera/src/components/mobile_view/reducer.ts

```diff
@@ -54,4 +55,34 @@ export function arReducer(
     default:
       return state;
   }
+}
+
+const SET_REFRESHABLE = 'SET_REFRESHABLE';
+export function dispatchSetRefreshable(canRefresh: boolean) {
+  return {type: SET_REFRESHABLE, payload: canRefresh};
+}
+
+const SET_FORCE_POST = 'SET_FORCE_POST';
+export function dispatchSetForcePost(forcePost: boolean) {
+  return {type: SET_FORCE_POST, payload: forcePost};
+}
+
+export const getRefreshable = (state: State) =>
+    state.entities.mobile.isRefreshable;
+export const getForcePost = (state: State) => state.entities.mobile.forcePost;
+
+export function mobileReducer(
```

Comment:
Added to state so that refreshable changes can interact with the displaying the refresh button in the model viewer preview.